src/PIL/ImageGrab.py

```diff
@@ -62,6 +61,18 @@ def grab(bbox=None, include_layered_windows=False, all_screens=False, xdisplay=N
                 left, top, right, bottom = bbox
                 im = im.crop((left - x0, top - y0, right - x0, bottom - y0))
             return im
+        elif not Image.core.HAVE_XCB and shutil.which("gnome-screenshot"):
```

Comment:
I'm not sure `HAVE_XCB` should be required here. This is only set if Pillow is compiled with XCB, but it says nothing about whether the display supports screenshots. Anyone installing Plilow from a wheel would be unable to use gnome-screenshot. Linux users can still force XCB to be used even with gnome-screenshot present by passing `xdisplay=''`. I think release notes/documentation should be sufficient here.

```suggestion
        elif shutil.which("gnome-screenshot"):
```

Reply:
Ok, I've added a commit for documentation.